man/abireport.1

```diff
@@ -1,115 +0,0 @@
-.\" generated with Ronn/v0.7.3
```

Comment:
This file is now empty. Can it be removed from the tree instead?

Reply:
Both of those were accidental. I am happy to remove them from the tree though since they should be generated for a release instead.